src/core/server/base.js

```diff
@@ -115,6 +116,14 @@ function baseServer(routes, createStore, { appInstanceName = appName } = {}) {
   const app = new Express();
   app.disable('x-powered-by');
 
+  const sentryDsn = config.get('sentryDsn');
+  if (sentryDsn) {
+    Raven.config(sentryDsn).install();
+    app.use(Raven.errorHandler());
+  } else {
+    log.warn('Sentry reporting is disabled because sentryDsn was falsey');
```

Comment:
"was falsey" seems odd to write in a log. Could this be a bit more instructive?

`'Sentry reporting is disabled. Set config.sentryDsn to enable reporting.'`